app/assets/locales/locale-ja.json

```diff
@@ -1641,7 +1650,7 @@
         "from": "From",
         "header": "転送の詳細設定",
         "header_subheader": "転送機能は他のBitSharesアカウントに資金を送金するために使用されます。",
-        "memo": "メモ",
+        "memo": "最後の20のログエントリ",
```

Comment:
I did not see that before. Please refactor that you do not use news.memo (since this is used elsewhere as well).

Please create a proper entry within your new report section (for all languages)